integration/spark/app/build.gradle

```diff
@@ -40,10 +40,16 @@ ext {
     versions = versionsMap[shortVersion]
 }
 
-configurations.all { // https://github.com/apache/spark/pull/38355 - can be remove for Spark 3.3.2
-    resolutionStrategy {
-        // https://github.com/FasterXML/jackson-databind/issues/3627
-        force "com.fasterxml.jackson:jackson-bom:$jacksonVersion"
+// This is done, in order to prevent Jackson version conflicts during the tests in 'app'
```

Comment:
When commenting this out, the integration test for 3.5 is passing. Perhaps this one also replaces Jackson dependency within Spark internals and loads Spark with a Jackson version that does not match.